pkg/sql/sem/tree/schedule.go

```diff
@@ -33,9 +38,15 @@ var _ Statement = &ScheduledBackup{}
 func (node *ScheduledBackup) Format(ctx *FmtCtx) {
 	ctx.WriteString("CREATE SCHEDULE")
 
-	if node.ScheduleLabel != nil {
-		ctx.WriteString(" ")
-		ctx.FormatNode(node.ScheduleLabel)
+	if node.ScheduleLabelSpec != nil {
```

Comment:
Following from the comment above, we can remove this nil check if we make ScheduleLabelSpec a non-pointer.

Reply:
done